tasks/setup.yaml

```diff
@@ -7,4 +7,4 @@ tasks:
 
       - description: "Initialize the cluster with Zarf"
         # renovate: datasource=github-tags depName=defenseunicorns/init versioning=semver
-        cmd: "zarf package deploy oci://defenseunicorns/init:v0.31.1-${UDS_ARCH} --confirm"
+        cmd: "zarf package deploy oci://defenseunicorns/init:v0.31.4-${UDS_ARCH} --confirm"
```

Comment:
Zarf v0.32+ does not use arch in OCI ref anymore, this will break soon

Reply:
yeah we were waiting on UDS CLI, which wasn't compatible with Zarf 0.32